src/steps/node.rs

```diff
@@ -53,17 +63,21 @@ pub fn run_npm_upgrade(_base_dirs: &BaseDirs, run_type: RunType) -> Result<()> {
         let uid = Uid::effective();
 
         if metadata.uid() != uid.as_raw() {
+            if metadata.uid() == 0 && (ctx.config().npm_use_sudo()) {
+                use_sudo = true;
+            }
+
             return Err(SkipStep(format!(
-                "NPM root at {} is owned by {} which is not the current user",
-                npm_root.display(),
-                metadata.uid()
-            ))
-            .into());
+                    "NPM root at {} is owned by {} which is not the current user. Set use_sudo = true under the NPM section in your configuration to run NPM as sudo",
+                    npm_root.display(),
+                    metadata.uid()
+                ))
+                    .into());
```

Comment:
Shouldn't this be in an else-block? This way the previous `if` is just ignored.

Reply:
My bad. Pushed a fix